client/site-profiler/components/domain-information/index.tsx

```diff
@@ -46,53 +48,72 @@ export default function DomainInformation( props: Props ) {
 		} );
 	};
 
+	const fWhois: { [ name: string ]: boolean } = {};
```

Comment:
Can we just name this `filteredWhois` or `withoutRedactedWhois`..?

Reply:
Done, thanks.